Backend/app/auth/auth_service.py

```diff
@@ -43,21 +43,18 @@
 
 
 def create_access_token(data: dict[str, str], expires_delta: timedelta | None = None) -> str:
-    """Create a jwt token from data with a expire date
+    """Creates a JWT access token with given data and expiration.
 
     Args:
-    ----
-        data (dict): Info to be stored in the token
-        expires_delta (timedelta | None, optional): Expire date of the token
-
-    Raises:
-    ------
-        CreateJWTException: if an error ocurred creating JWT Token
+       data: Dictionary of data to encode in the token.
```

Comment:
We don't wanna lose type hints in docstrings. I use VSCODE with this [extension](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) that handles docstrings automatically.

Reply:
Got it, reverting...